assets/training/finetune_acft_image/components/finetune/mmt/spec.yaml

```diff
@@ -0,0 +1,341 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+version: 0.0.1
+name: mmtracking_video_multi_object_tracking_finetune
+display_name: Image Tracking MMTracking Model Finetune
```

Comment:
display_name and description should be updated to reflect mot specific content.

Reply:
changed. thanks for reminder.